### PR TITLE
T5 testnet

### PR DESCRIPTION
**Pull request by roman-khimov**
Syncs well with compatible stateroots as of now.

### CODE REVIEW

File: config/protocol.testnet.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ProtocolConfiguration:
   Magic: 877933390
+  MaxBlockSystemFee: 150000000000
```
Comment (AnnaShaleva) [inline config/protocol.testnet.yml:3]: No `MaxBlockSize` change? It differs from the default.

File: config/protocol.testnet.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ProtocolConfiguration:
-  Magic: 877933390
+  Magic: 894710606
```
Comment (AnnaShaleva) [inline config/protocol.testnet.yml:2]: `TestGetConfigFromContext` should be adjusted wrt this change.

File: config/protocol.testnet.yml (hunk 1 above)
Comment (roman-khimov) [reply config/protocol.testnet.yml:4]: Yeah, fixed.

File: config/protocol.testnet.yml (hunk 2 above)
Comment (roman-khimov) [reply config/protocol.testnet.yml:2]: Yep, fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**